helpers/useData.ts

```diff
@@ -0,0 +1,44 @@
+import { useAsync, UseAsyncReturn } from "react-async-hook";
+import {
+  Database,
+  NamedSchema,
+  Schema,
+  StoreNames,
+  StoreValue
+} from "remultiform/database";
+
+import useDatabase from "./useDatabase";
+import { DatabaseContext } from "remultiform/dist/esm/database-context";
+import getProcessRef from "./getProcessRef";
+
+const useData = <
+  DBSchema extends NamedSchema<string, number, Schema>,
+  StoreName extends StoreNames<DBSchema["schema"]>
+>(
+  context: DatabaseContext<DBSchema> | undefined,
+  storeName: StoreName
+): UseAsyncReturn<
+  StoreValue<DBSchema["schema"], StoreName> | undefined,
+  [Database<DBSchema> | undefined, boolean, StoreName, string | undefined]
+> => {
+  const database = useDatabase(context);
+  const processRef: string | undefined = getProcessRef();
```

Comment:
This explicit typing shouldn't be necessary. Removing it isn't a blocker, though.